lib/ansible/modules/cloud/azure/azure_rm_webapp.py

```diff
@@ -708,11 +708,12 @@ def exec_module(self, **kwargs):
                 if self.purge_app_settings:
                     to_be_updated = True
                     self.app_settings_strDic.properties = dict()
+                    self.to_do = Actions.UpdateAppSettings
 
                 # check if app settings changed
                 if self.purge_app_settings or self.is_app_settings_changed():
                     to_be_updated = True
```

Comment:
so is it possible to update webapp and app settings at once?
seems like UpdateAppSettings overrides CreateOrUpdate, so if user wants to update both, only  settings will be updated